server/main_server/src/main.cpp

```diff
@@ -494,34 +493,11 @@ Environment Variables:
                 return true;
             }; // do_validate
 
-            // Validate the server configuration. If that succeeds, move on to validating the
-            // irods_environment.json file.
-            const auto schema_dir = irods::get_irods_home_directory() /
-                                    fmt::format("configuration_schemas/v{}", IRODS_CONFIGURATION_SCHEMA_VERSION);
+            // Validate the server configuration.
+            const auto path_suffix = fmt::format("configuration_schemas/v{}", IRODS_CONFIGURATION_SCHEMA_VERSION);
+            const auto schema_dir = irods::get_irods_home_directory() / path_suffix;
```

Comment:
Remove any documentation that talks about validating the service account's irods_environment.json file.

Reply:
Linked this comment to issue #8012.